### ISSUE TITLE
Device exposes too many controls?

### ISSUE THREAD

**Issue by chrisvanpatten** [pschroeder89/homebridge-levoit-humidifiers#19]
**Describe The Bug:**
Since upgrading to the newest version of the humidifier plugin, my humidifiers now expose multiple "humidifier" controls in the Home app. It's not totally clear what the difference is between the controls.

Additionally, the "Cool Mist" control also seems confusing, as it's not clear if that corresponds to fan speed or some other element.

If the controls actually serve separate purposes it might be an opportunity to give them clearer labels.

**To Reproduce:**
Use latest version of plugin with a Classic300s Levoit humidifier.

**Expected behavior:**
One set of humidity controls, or better labels that make the differences clear.

**Logs:**

no logs

**Plugin Config:**

normal config

**Screenshots:**

![humidifier](https://user-images.githubusercontent.com/1231306/158080688-4c383e87-c177-4e3c-996e-dada4c95d3e3.png)

**Environment:**

* **Plugin Version**: 1.6.0
* **Homebridge Version**: 1.4.0
* **Node.js Version**: 14.19.0
* **NPM Version**: 6.14.16
* **Operating System**: Raspbian Buster

**Comment by pschroeder89** [comment]
Hey @chrisvanpatten, it looks like you have a cached version of the previous plugin. Only one Humidifier control should be exposed. You can clear your device cache in Homebridge and it should fix it. If it doesn't, please try uninstalling and reinstalling the plugin, and also let me know. 

I saw one report of someone needing to uninstall and reinstall the plugin, which is not a behavior I want people to have to do, so if you need to, I'd like to know. Thanks!

As for the Cool Mist control, it is essentially "manual mode". The explanation of each toggle can be found in the [README](https://github.com/pschroeder89/homebridge-levoit-humidifiers#features)

Let me know if anything is unclear and I can update the readme or answer here.

**Comment by chrisvanpatten** [comment]
Thanks @pschroeder89 appreciate the response!

I'll give this a shot and report back.


On a related note, it might be cool if it were possible, through configuration, to have the ability to configure which controls are exposed to HomeKit, e.g.

```
"controls": {
    "Cool Mist": false,
    "Display": false,
    "Night Light": false
}
```

**Comment by pschroeder89** [comment]
@chrisvanpatten this is actually my plan for 1.7.0 (having toggles in the config). I plan to have a beta out by next weekend. I'll let you know so you can test it out. Thanks!

**Comment by chrisvanpatten** [comment]
@pschroeder89 Awesome, that sounds great.

Also just to confirm, I was able to get the correct number of devices showing up by flushing my device cache in Homebridge, so I think my original issue here can be closed out!

**Comment by pschroeder89** [comment]
@chrisvanpatten Awesome! Thanks for testing. Also, I just published 1.7.0-beta1, with support for turning off accessories for the plugin. Example config (you can omit the warm mist option for the Classic300s):
```
{
  "platforms": [
    {
      "name": "Levoit Humidifiers",
      "email": "email",
      "password": "password",
      "platform": "LevoitHumidifiers",
      "accessories": {
         "display": false, 
         "sleep_mode": false, 
         "cool_mist": false, 
         "warm_mist": false,
         "night_light": false
      }
    }
  ]
}
```

Feel free to mention any observations / bugs in the discussion thread here: https://github.com/pschroeder89/homebridge-levoit-humidifiers/discussions/25
I've added specific logic to prevent needing to flush the cache when changing the accessories values, so if you see lingering accessories after making a change, please let me know.

Thanks again for testing and thanks a lot for the donation!
